src/CSV_Parser.php

```diff
@@ -292,6 +331,29 @@ public function parse($headers = true) {
 	}
 
 	/**
+	 * Convert character encoding of data
+	 * @throws Jabran\Exception\InvalidEncodingException
+	 * @return self
+	 */
+	public function convertEncoding() {
```

Comment:
A check might be overkill since PHP always sets a default and now we set one in the constructor.